gengo-v2/generator/generator.go

```diff
@@ -32,8 +32,8 @@ type Target interface {
 	Name() string
 	// Path returns the package import path.
 	Path() string
-	// SourcePath returns the location of the package on disk.
-	SourcePath() string
+	// Dir returns the location of the package on disk.
+	Dir() string
```

Comment:
Based on the commit comment ("SourcePath" is ambiguous - it's not the source, but the destination. But it will hold the outputted source code.)

Would it be appropriate to include "output", "target" or "destination" in the identifier or in the godoc?

Reply:
ACK